### PR TITLE
[api] for the debugger: get_raw_state_value

### PR DESCRIPTION
**Pull request by msmouse**
To support getting resource groups and state metadata.

### CODE REVIEW

**Review by runtian-zhou — COMMENTED** [review]
Thank you! Would love to see an api like that as long as others agree on this.

**Comment by grao1991** [comment]
Shell we expose this as a "debug API" and guard it behind some flags, and add a note saying it's not considered as part of our "stable API".  I want to leave some flexibility here.

**Comment by msmouse** [comment]
@banool  is there a way to mark an endpoint "not promised, do not use"?

**Comment by davidiw** [comment]
> @banool is there a way to mark an endpoint "not promised, do not use"?

if there isn't one, I might recommend we just put it on something like host/experimental/API in contrast to host/v1/API

File: api/src/state.rs [hunk 1]
```diff
@@ -214,6 +214,43 @@ impl StateApi {
...
+    /// in the request body.
+    ///
+    /// The Aptos nodes prune account state history, via a configurable time window.
+    /// If the requested ledger version has been pruned, the server responds with a 410.
+    #[oai(
+        path = "/state_values/raw",
+        method = "post",
+        operation_id = "get_raw_state_value",
+        tag = "ApiTags::StateValues"
+    )]
```
Comment (banool) [inline api/src/state.rs:230]:
```suggestion
        tag = "ApiTags::StateValues",
        hidden
    )]
```

**Review by banool — COMMENTED** [review]
There is no way to mark it as "do not use" but if we upgrade our version of poem (https://github.com/aptos-labs/aptos-core/pull/8197) it is possible to hide an endpoint. I've left a suggestion in the code.

**Comment by msmouse** [comment]
Thanks @banool. I added "hidden" and changed the tag. PTAL

**Comment by msmouse** [comment]
moved to `/experimental/stata_values/raw`

**Review by grao1991 — APPROVED** [review]

**Review by areshand — APPROVED** [review]

**Review by banool — APPROVED** [review]
Looks good, nice nice.

**Pull request merged**